Migrations/ChefsterDbContextModelSnapshot.cs

```diff
@@ -16,30 +15,59 @@ partial class ChefsterDbContextModelSnapshot : ModelSnapshot
         protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
-            modelBuilder
-                .HasAnnotation("ProductVersion", "8.0.6")
-                .HasAnnotation("Relational:MaxIdentifierLength", 64);
+            modelBuilder.HasAnnotation("ProductVersion", "8.0.6");
 
-            MySqlModelBuilderExtensions.AutoIncrementColumns(modelBuilder);
+            modelBuilder.Entity("Chefster.Models.AddressModel", b =>
```

Comment:
Do we need the migrations since we can just add the column in MySql Workbench?

Reply:
I was having local db issues and had to delete the entire Migrations folder for it to work. we will add the column to the production db via MySQL workbench like you mentioned. moving forward it might be best for use to continue to check in the migrations so that we can share them and ensure that our local dbs match up